### DISCUSSION TITLE
It would be nice if the Layer representing a Component with {draggable: false} didn't show the 'move' icon

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5943] [General]
Just that.

**Comment by mosh-tudor** [comment]
## This is already implemented:


<img width="229" alt="Screenshot 2024-06-14 at 20 00 47" src="https://github.com/GrapesJS/grapesjs/assets/668451/23d63559-0724-4140-9e9a-5d37ff902593">

**Reply by glaprida** [reply]
Which version are you on? I thought it was implemented... but I recently updated to version 0.21.10 and now I see the move icon on every layer. (not sure when it started happening, it might not be related to the update).


<img width="418" alt="Screenshot 2024-06-14 at 15 05 20" src="https://github.com/GrapesJS/grapesjs/assets/10442382/2c1798d7-e1a4-4aba-9e21-e631f760b102">

the one that says "Cue..." is this component below (with `{draggable:false}` )

<img width="567" alt="Screenshot 2024-06-14 at 15 04 55" src="https://github.com/GrapesJS/grapesjs/assets/10442382/14942712-1781-45a6-9fd4-4aa43ceea264">

In fact, all of those components shown in the first image are set to draggable false, and they all show the arrow.

Im using the default layerManager, only with appendTo.

**Reply by artf** [reply]
looks like a regression...

Update: fixed [here](https://github.com/GrapesJS/grapesjs/commit/f20d7d3b18f92daba16bf18ed18a11f57f38ed1d)
